src/common.rs

```diff
@@ -0,0 +1,64 @@
+pub(crate) use std::{
+  convert::Infallible,
+  env,
+  ffi::OsString,
+  fs::{self, FileType},
+  future,
+  future::Future,
+  io::{self, Write},
+  mem::MaybeUninit,
+  net::{SocketAddr, ToSocketAddrs},
+  path::{Path, PathBuf},
+  pin::Pin,
+  str,
+  sync::Arc,
+  task::{Context, Poll},
+  time::Duration,
+};
+
+pub(crate) use ::{
+  agora_lnd_client::Millisatoshi,
+  futures::{
+    future::{BoxFuture, OptionFuture},
+    FutureExt, Stream, StreamExt,
+  },
+  http::uri::Authority,
+  hyper::{
+    header::{self, HeaderValue},
+    server::conn::AddrIncoming,
+    service::Service,
+    Body, Request, Response, StatusCode,
+  },
+  lexiclean::Lexiclean,
+  maud::Markup,
+  serde::Deserialize,
+  snafu::{IntoError, ResultExt},
+  structopt::StructOpt,
+  tokio::{
+    io::{AsyncReadExt, AsyncWriteExt},
+    task,
```

Comment:
I wonder whether using `task::spawn` would be a bit confusing, since there's also `std::task` and `std::thread::spawn`. Maybe `tokio::task::spawn` would be better.